### PR TITLE
cluster-ui: add uplot library

### PR DESCRIPTION
**Pull request by xinhaoz**
Part of #74516

This commit adds the uplot library to the cluster-ui package.

Release note: None

### CODE REVIEW

**Review by jocrl — APPROVED** [review]
:lgtm: 

> *[Reviewable](https://reviewable.io/reviews/cockroachdb/cockroach/80195#-:-N02kUyd4FJcVoQPPAzt:bg7igkv)* status: :shipit: complete! 1 of 0 LGTMs obtained

**Comment by xinhaoz** [comment]
TFTR! Will merge vendor changes after this lands :)
bors r=jocrl

**Pull request merged**
